### PR TITLE
Add link to the Spectrum service

### PR DESCRIPTION
**Pull request by bogdal**

### CODE REVIEW

**Comment by rafalp** [comment]
Thanks! Could you also create issue template? Something like:

```
Thank you for contributing to Ariadne!

Please take note that we are only using GitHub issues for the roadmap, bugs and general issues with the project's codebase.

For usage questions or feedback, please use our Spectrum instead: https://spectrum.chat/ariadne)

Thank you!
```

**Comment by bogdal** [comment]
@rafalp I have configured GH templates but in a different way. Check this out:
image

File: .github/ISSUE_TEMPLATE/issue_or_bug_report.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: ✏ Issue or bug report
+about: Use for general issues and bug reports related only to the project's codebase.
```
Comment (rafalp) [inline .github/ISSUE_TEMPLATE/issue_or_bug_report.md:3]:
```suggestion
about: Use for bug reports or other issues with the codebase.
```

**Comment by rafalp** [comment]
@NyanKiyoshi yup, we could also include something like this near the bottom of readme:

> Also make sure you follow [@AriadneGraphQL](https://twitter.com/AriadneGraphQL) on Twitter for latest updates, news and random musings!

**Comment by bogdal** [comment]
@NyanKiyoshi @rafalp What do you think guys?
image

**Comment by rafalp** [comment]
Let's not add Twitter to this screen, its absolute garbage for gathering knowledge, and I would like to be able to easily find and link to past discussions when people ask questions ;)

**Comment by bogdal** [comment]
So we are ready to go now 🚀

**Pull request merged**